app/src/main/java/com/example/nguyenthanhtungh/moviedb/MyApplication.kt

```diff
@@ -1,6 +1,22 @@
 package com.example.nguyenthanhtungh.moviedb
 
 import android.app.Application
+import com.example.nguyenthanhtungh.moviedb.di.apiModule
+import com.example.nguyenthanhtungh.moviedb.di.appModule
+import com.example.nguyenthanhtungh.moviedb.di.repositoryModule
+import com.example.nguyenthanhtungh.moviedb.di.viewModelModule
+import org.koin.android.ext.android.startKoin
 
 class MyApplication : Application() {
```

Comment:
Ô đã khai báo arrtribute  name trong manifest chưa?

Reply:
(y), thank you